src/extension/src/ActionHandler.py

```diff
@@ -196,21 +192,14 @@ def uninstall(self):
             return Constants.ExitCode.HandlerFailed
 
         finally:
-            if stdout_file_mirror is not None:
-                stdout_file_mirror.stop()
-            if file_logger is not None:
-                file_logger.close()
+            self.close_file_logger(file_logger, stdout_file_mirror)
 
     def enable(self):
         stdout_file_mirror = None
         file_logger = None
-        try:
-            log_file_name = datetime.datetime.utcnow().strftime("%Y-%m-%d_%H-%M-%S") + "_Enable"
-            file_logger = self.utility.create_log_file(self.ext_env_handler.log_folder, log_file_name)
-            if file_logger is not None:
-                self.logger.file_logger = file_logger
-                stdout_file_mirror = StdOutFileMirror(file_logger)
 
+        try:
+            file_logger, stdout_file_mirror = self.setup_file_logger(action="Enable")
```

Comment:
To be discussed this week as in the outdated comment. Please set up half an hour to discuss with your proposal on how to mitigate log overflows from repeated re-enables (on the same sequence number).